src/_gettsim_tests/test_renten_alter.py

```diff
@@ -42,14 +43,61 @@
 
 @pytest.fixture(scope="module")
 def input_data():
-    file_name = "renten_alter.csv"
-    out = pd.read_csv(TEST_DATA_DIR / file_name)
+    # Original Data Source: Seibold, A. (2021).
+    # Reference points for retirement behavior:
+    # Evidence from german pension discontinuities.
+    # American Economic Review, 111(4), 1126-65.
+
+    # IDs were added to the original data
+    # m_pflichtbeitrag is added as an identifier for pension kind
+    # 60: Regelaltersrente, 420: Langjährig Versichert, 180: Altersrente für Frauen
+    original_data = pd.read_csv(TEST_DATA_DIR / "renten_alter.csv")
+
+    original_data.columns = original_data.columns.str.lower()
+
+    # Rename variables to GETTSIM convention
+    var_names = {
+        "id": "p_id",
+        "yob": "geburtsjahr",
+        "mob": "geburtsmonat",
+        "regsray": "Rentenalter_Jahr",
+        "regsram": "Rentenalter_Monat",
+        "m_pflichtbeitrag": "m_pflichtbeitrag",
+    }
+    out = original_data[[*var_names]].rename(columns=var_names).copy()
+
+    # Create IDs
+    out["tu_id"] = out["p_id"]
+    out["hh_id"] = out["p_id"]
+
+    # Create variables needed in GETTSIM
+    out["weiblich"] = True
+    out["jahr"] = random.choice(YEARS)
+    out["alter"] = out["jahr"] - out["geburtsjahr"]
+    out["m_arbeitsunfähig"] = 0
+    out["m_krank_ab_16_bis_24"] = 0
+    out["m_mutterschutz"] = 0
+    out["m_arbeitslos"] = 0
+    out["m_ausbild_suche"] = 0
+    out["m_schul_ausbild"] = 0
+    out["m_alg1_übergang"] = 0
+    out["m_geringf_beschäft"] = 0
+    out["m_freiw_beitrag"] = 0
+    out["m_ersatzzeit"] = 0
+    out["m_kind_berücks_zeit"] = 0
+    out["m_pfleg_berücks_zeit"] = 0
+    out["_ges_rente_altersgrenze_abschlagsfrei"] = out["Rentenalter_Jahr"] + out[
+        "Rentenalter_Monat"
+    ] * (1 / 12)
+    out.loc[out["m_pflichtbeitrag"] == 180, "y_pflichtbeitr_ab_40"] = 10
```

Comment:
Can you add a comment before those two lines, what is done and what the assumption is?